src/lib/Components/Consignments/Screens/ConsignmentsArtist.tsx

```diff
@@ -41,31 +42,38 @@ export default class Artist extends React.Component<Props, State> {
     this.props.navigator.pop()
   }
 
-  textChanged = async (text: string) => {
+  textChanged = (text: string) => {
     this.setState({ query: text, searching: text.length > 0 })
     this.searchForQuery(text)
   }
 
-  // TODO: Add throttling
-  searchForQuery = async (query: string) => {
-    const results = await fetchQuery<ConsignmentsArtistQuery>(
+  // tslint:disable:member-ordering
+  searchForQuery = throttle(async (query: string) => {
+    const data = await fetchQuery<ConsignmentsArtistQuery>(
       environment,
       graphql`
         query ConsignmentsArtistQuery($query: String!) {
-          matchArtist(term: $query) {
-            internalID
-            name
-            image {
-              url
+          searchConnection(query: $query, first: 10, entities: [ARTIST], mode: AUTOSUGGEST) {
```

Comment:
Sneaky! This is a good a time as any to clarify the difference here.

@cavvia , we have 2 ways of getting what seems to be the same/v similar data. One is to use `/match/suggest?indexes[]=Artist` (global auto suggest restricted to just artists), and the other is to use `/match/artists`.

It's hard for me to understand the difference between the two, the only clear explicit difference seems to be that the auto-suggest one searches alternate names as well. But there might be other functional differences lurking beneath the hood.

Reply:
I would expect suggest to use the auto-suggest query so it's designed for completion of an active query (which is being typed out) rather than a finalised query term.

Reply:
Interesting! I feel like all the use cases I've seen of `/match/artists` have generally been for auto-completes. But we can revisit if it comes up again!